experiments/stv/notebooks/astroquery.json

```diff
@@ -0,0 +1 @@
+﻿{"paragraphs":[{"text":"%python\n# get the 2MASS photometry for the set of examples of good astrometric solutions (uses GACS call-out for DPAC crossmatch at GDR2)\n#\n# TODO: replace this with an internal GEDR3/2MASS match (using AXS) when deployed.\n\nimport warnings\nwarnings.filterwarnings(\"ignore\")\n\nfrom astroquery.utils.tap.core import TapPlus\ngaia = TapPlus(url=\"https://gea.esac.esa.int/tap-server/tap\")\n\nadql_query = \"SELECT TOP 10 * from gaiadr2.gaia_source\"\n# ... last three predicates exclude low Galactic latitudes and the Magellanic Clouds.\n\n# execute the query\njob = gaia.launch_job(adql_query)\nresults = job.get_results()\n","user":"gaiauser","dateUpdated":"2020-10-13T17:36:59+0000","config":{"colWidth":12,"fontSize":9,"enabled":true,"results":{},"editorSetting":{"language":"python","editOnDblClick":false,"completionSupport":true,"completionKey":"TAB"},"editorMode":"ace/mode/python"},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1602610577677_-1688817092","id":"20201013-173617_1062770921","dateCreated":"2020-10-13T17:36:17+0000","dateStarted":"2020-10-13T17:36:59+0000","dateFinished":"2020-10-13T17:37:03+0000","status":"FINISHED","errorMessage":"","progressUpdateIntervalMs":500,"focus":true,"$$hashKey":"object:3435"},{"text":"%python\nprint(results)","user":"gaiauser","dateUpdated":"2020-10-13T17:38:05+0000","config":{"colWidth":12,"fontSize":9,"enabled":true,"results":{},"editorSetting":{"language":"python","editOnDblClick":false,"completionSupport":true},"editorMode":"ace/mode/python"},"settings":{"params":{},"forms":{}},"apps":[],"jobName":"paragraph_1602610619050_-332377569","id":"20201013-173659_1847815768","dateCreated":"2020-10-13T17:36:59+0000","dateStarted":"2020-10-13T17:38:05+0000","dateFinished":"2020-10-13T17:38:05+0000","status":"FINISHED","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:3436"}],"name":"experiments/stv/astroquery","id":"2FND9GEZ9","noteParams":{},"noteForms":{},"angularObjects":{"spark::2FND9GEZ9":[],"python:shared_process":[]},"config":{"isZeppelinNotebookCronEnable":false,"looknfeel":"default","personalizedMode":"false"},"info":{}}
```

Comment:
Can we run JSON files through a formatting tool to make them human readable (and editable) before we check them into git.

Reply:
Yep, todo for me to modify the existing JSON files to make them human readable